torch/onnx/utils.py

```diff
@@ -968,6 +968,12 @@ def _validate_dynamic_axes(dynamic_axes, model, input_names, output_names):
                     value_dict[x] = str(key) + '_dynamic_axes_' + str(i + 1)
             dynamic_axes[key] = value_dict
 
+def _add_block(node, input_node, op_name, **kwargs):
```

Comment:
`addNode` returns the newly added node, so it can be called iteratively to add multiple nodes to a block.
Is this what you meant?

---
In reply to: [442365121](https://github.com/pytorch/pytorch/pull/38476#discussion_r442365121) [](ancestors = 442365121)

Reply:
Will the input/output relationship be preserved by adding multiple consecutive nodes within a subblock? Can we support multiple input outputs in that case?

Reply:
We can leave this for now since it's a non-blocking issue and not needed currently. 
We will need to get back to updating this function for more general cases when needed.